src/EVM/Types.hs

```diff
@@ -768,6 +769,7 @@ data Block = Block
 data Contract = Contract
   { code        :: ContractCode
   , storage     :: Expr Storage
+  , tStorage    :: Expr Storage
```

Comment:
same here, maybe `transientStorage`?

Reply:
I'm resolving, will ask dxo and then fix accordingly.